### ISSUE TITLE
No Searchbar

### ISSUE THREAD

**Issue by imrohitkumar** [YassKnight/LimeBrowser#1]
Hey, I liked your Browser but it has some major problems.
1. No searchbar, how should i suppose to search because every browser provides a searchbar
2. No settings
3. No Bookmarks
4. No History
5. Black Bar on Top is useless

**Comment by YassKnight** [comment]
> Hey, I liked your Browser but it hase some major problems.
> 
> 1. No searchbar, how should i suppose to search because every browser provides a searchbar
> 2. No settings
> 3. No Bookmarks
> 4. No History
> 5. Black Bar on Top is useless



Yeah, in order to meet the needs of the organization for the time being, functions will be added gradually in the future. Thank you for your suggestion

**Comment by imrohitkumar** [comment]
Yeah, I Would love using your browser when it's ready with all features.
How much time would it take normally ?
 can i suggest you some good projects on github like yours [but they are dead]

**Comment by YassKnight** [comment]
> Yeah, I Would love using your browser when it's ready with all features.
> How much time would it take normally ?
> can i suggest you some good projects on github like yours [but they are dead]

It's my pleasure that you like this project. I would like to see some good projects.
But I can’t determine the time, maybe a month or two, because I have other projects, but I'll open a branch and try to perfect it as much as I can

**Comment by imrohitkumar** [comment]
Hey, Thanks for the quick response here are some projects Which i think can be useful for you
1. https://github.com/ricky9090/EasyBrowser [my favorite]
2. https://github.com/mengkunsoft/Mkbrowser
3. https://github.com/yangjufo/Editable-Android-Browser
As of now i am also learning developing apps in java and kotlin, would love contributing to your project in future.
